deployment/deployutils/deployutils.cpp

```diff
@@ -827,6 +827,12 @@ class CGenerateJSFromXSD
               LoadComboBox("Software/DaliServerProcess", bAddBlank, m_pEnv, m_pEnv, strBuf);
               extraInfo = strBuf.str();
             }
+            else if (strcmp(type, "addrCleanerPluginType")==0)
```

Comment:
Would be more generic if you tested for something like "type ends with PluginType" and then generated the xpath from by substituting PluginType for PluginProcess...